app/main.py

```diff
@@ -1,13 +1,28 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
```

Comment:
please, use code style as mentioned in this checklist:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md
Here check this rule:
`The curly braces must be located in one of two options: open and start with the text or have a line break between the text.`